doc/subsystems/networking/qemu_setup.rst

```diff
@@ -133,18 +135,26 @@ To stop the daemons, press Ctrl+C in the corresponding terminal windows
 (you need to stop both ``loop-slip-tap.sh`` and ``loop-socat.sh``).
 
 
-Setting up Zephyr and NAT/masquerading on QEMU host to access Internet
-**********************************************************************
+.. _networking_internet:
 
-To access the internet from a Zephyr application using IPv4,
+Setting up Zephyr and NAT/masquerading on host to access Internet
+*****************************************************************
+
+To access the Internet from a Zephyr application, some additional
+setup on the host may be required. This setup is common for both
+application running in QEMU and on real hardware, assuming that
+a development board is connected to the development host (if a
+board is connected to a dedicated router, it should not be needed).
+
+To access the Internet from a Zephyr application using IPv4,
 a gateway should be set via DHCP or configured manually.
-For applications using the :ref:`net_app_api` facility (with the config option
-:option:`CONFIG_NET_APP` enabled),
+For applications using the "Settings" part of :ref:`net_app_api`
+facility (with the config option :option:`CONFIG_NET_APP_SETTINGS` enabled),
 set the :option:`CONFIG_NET_APP_MY_IPV4_GW` option to the IP address
 of the gateway. For apps not using the :ref:`net_app_api` facility, set up the
 gateway by calling the :c:func:`net_if_ipv4_set_gw` at runtime.
 
-To access the internet from a custom application running in QEMU, NAT
+To access the Internet from a custom application running in QEMU, NAT
```

Comment:
lowercase ``internet`` was fine (it's not a proper name)

Reply:
I thought it's the norm to write it capitalized, at least I saw it written it like that in a bunch of newspaper/magazine articles, etc. But ok, reverting back.

Reply:
Note that in the heading of this section we had it capitalized, while in the text - not ;-).